src/Spago/Command/Bundle.purs

```diff
@@ -35,7 +41,7 @@ type RawBundleOptions =
   , extraArgs :: Array String
   }
 
-run :: forall a. Spago (BundleEnv a) Unit
+run :: ∀ a. Spago (BundleEnv a) Unit
```

Comment:
I reorganized this function a little to reduce noise, I hope it's ok.

Reply:
Yes, it looks great!